jobs/orcid/DepositOrcidReview.php

```diff
@@ -19,55 +19,63 @@
 use APP\core\Application;
 use APP\facades\Repo;
 use APP\orcid\OrcidReview;
-use APP\submission\Submission;
 use Carbon\Carbon;
 use GuzzleHttp\Exception\ClientException;
 use PKP\config\Config;
-use PKP\context\Context;
 use PKP\jobs\BaseJob;
 use PKP\orcid\OrcidManager;
-use PKP\submission\reviewAssignment\ReviewAssignment;
 
 class DepositOrcidReview extends BaseJob
 {
     public function __construct(
-        private Submission $submission,
-        private Context $context,
-        private ReviewAssignment $reviewAssignment,
+        private int $submissionId,
+        private int $contextId,
+        private int $reviewAssignmentId
     ) {
         parent::__construct();
     }
 
     /**
      * @inheritDoc
      */
-    public function handle()
+    public function handle(): void
     {
+        $submission = Repo::submission()->get($this->submissionId);
+        $context = Application::getContextDAO()->getById($this->contextId);
+        $reviewAssignment = Repo::reviewAssignment()->get($this->reviewAssignmentId);
+
+        if (
+            $submission === null ||
+            $context === null ||
+            $reviewAssignment === null
+        ) {
```

Comment:
How about just including the review assignment ID, since the submission and context can be derived from that? Fewer sanity checks and less risk of inconsistent data sneaking through.